content/07.00.methods.md

```diff
@@ -341,9 +349,11 @@ Cell images were captured through fluorescent microscope EVOS for GFP signal det
 **Fluorescence-activated cell sorting (FACS).**
 Cells were immediately collected into 50ml tubes (From this point on, keep cells cold), and spun at 500 x g for 5min at 4oC.
 After DPBS wash, cell pellets were resuspended with FACS Sorting Buffer (1x DPBS without Ca2+/Mg2+, 2.5mM EDTA, 25mM HEPES, 1% BSA.
-The solution was filter sterilized, and kept at 4oC), pi-pet gently to make single cells.
+The solution was filter sterilized, and kept at 4oC), (pi-pet gently) with gentle pipetting to make single cells.
```

Comment:
can't tell what's happening here with the parentheses

Reply:
Me neither. I just copied/pasted what Sumei wrote. Maybe we can leave the "with gentle pipetting" without the part in parenthesis?

Reply:
```suggestion
The solution was filter sterilized, and kept at 4oC), with gentle pipetting to make single cells.
```